airflow/cli/commands/db_command.py

```diff
@@ -1,4 +1,13 @@
-# Licensed to the Apache Software Foundation (ASF) under one
+# Licensed to the Apache Software Foundation (ASF),
```

Comment:
Why did you put this code here? This is clearly in the wrong place. Please put this code in the right place